README.md

```diff
@@ -1 +1,54 @@
-# tribute
\ No newline at end of file
+# tribute
```

Comment:
Would please consider checking duplicated title in the README.md?
README.md should have one title